catelog.css

```diff
@@ -0,0 +1,274 @@
+@import url('https://vaspacx-ecommerce.netlify.app/main.css');
+
+body{
+    height:auto;
+}
+
+aside{
+    width: 12vw;
+    /* background-color: var(--secbg-color); */
```

Comment:
Hey remove the commented code and also add some comments on the code to enhance developer experience

Reply:
yes sure